File: intranet-angular.Server/Response/GrupoDeSlideResponse.cs
using intranet_angular.Server.Entities;

namespace intranet_angular.Server.Response
{
    public class GrupoDeSlideResponse
    {
        public int Id { get; set; }
        public int PaginaId { get; set; }
        public int? AutorId { get; set; }
        public string Autor { get; set; }
        public string AutorUrl { get; set; }
        public string Nome { get; set; }
        public int Posicao { get; set; }
        public DateTime DataPublicacao { get; set; } 
        // Relacionamento com Slides
        public ICollection<SlideResponse> Slides { get; set; }
    }
}

File: intranet-angular.client/src/response/grupoDeSlideResponse.ts
import { SlideResponse } from "./slideResponse";

export interface GrupoDeSlideResponse {
  id: number;
  paginaId: number;
  autorId: number;
  autor: string;
  dataPublicacao: Date;
  autorUrl: string;
  nome: string;
  posicao: number;
  slides: SlideResponse[]
}
